src/registrar/templates/domain_dnssec.html

```diff
@@ -0,0 +1,68 @@
+{% extends "domain_base.html" %}
+{% load static field_helpers url_helpers %}
+
+{% block title %}DNSSEC | {{ domain.name }} | {% endblock %}
+
+{% block domain_content %}
+
+  <h1>{% if dnssec_enabled %}Set up {% endif %}DNSSEC</h1>
+
+  <p>DNSSEC, or DNS Security Extensions, is additional security layer to protect your domain. Enabling DNSSEC ensures that when someone visits your domain, they can be certain that it's connecting to the correct server, preventing potential hijacking or tampering with your domain's records.</p>
+
+  <form class="usa-form usa-form--large" method="post">
+    {% csrf_token %}
+    {% if has_dnssec_records %}
+    <div class="usa-alert usa-alert--info usa-alert--slim margin-bottom-3">
+      <div class="usa-alert__body">
+      In order to fully disable DNSSEC on your domain, you will need to work with your DNS provider to remove your DNSSEC-related records from your zone.
+      </div>
+    </div>
+    <a
+      href="#toggle-dnssec-alert"
+      class="usa-button"
+      aria-controls="toggle-dnssec-alert"
+      data-open-modal
+      >Disable DNSSEC</a
+    >
+    {% elif dnssec_enabled %}
+    <div id="add-records">
+      <h2> Add DS Records </h2>
+      <p>In order to enable DNSSEC and add Delegation Signer (DS) records, you must first configure it with your DNS hosting service. Your configuration will determine whether you need to add DS Data or Key Data. Contact your DNS hosting provider if you are unsure which record type to add.</p>
+      <p>
+        <a href="{% url 'domain-dns-dnssec-dsdata' pk=domain.id %}" class="usa-button usa-button--outline">Add DS Data</a>
+        <a href="{% url 'domain-dns-dnssec-keydata' pk=domain.id %}" class="usa-button usa-button--outline">Add Key Data</a>
+        <button
+          type="submit"
+          class="usa-button usa-button--unstyled"
+          name="cancel_dnssec"
+          id="cancel_dnssec"
+      >Cancel</button>
+      </p>
+    </div>
+    {% else %}
+    <div id="enable-dnssec">
+      <div class="usa-alert usa-alert--info usa-alert--slim margin-bottom-3">
```

Comment:
Not sure if this is the right place to add this but noticing that the alert has the wrong padding, should be 46 and I'm seeing 56 on inspection. Ignore if this is just a USWDS issue.

Reply:
The USWDS slim alert with icon has a padding left of 3.5em

![Screen Shot 2023-10-13 at 11 37 14 AM](https://github.com/cisagov/getgov/assets/107004823/c196ab91-f389-480b-b746-2d75e550f1ff)

Overriding it will cause issues and require more refactoring. Ignoring.

![Screen Shot 2023-10-13 at 11 35 53 AM](https://github.com/cisagov/getgov/assets/107004823/5e1925b9-a21d-41e5-a269-6433a36a03eb)